Code/CryGame/Items/Weapons/Projectile.h

```diff
@@ -26,7 +26,7 @@ Description: Projectile
 #include "TracerManager.h"
 #include "Weapon.h"
 #include "AmmoParams.h"
-
+#include "CryGame/GameCVars.h"
```

Comment:
It would be better to move `IsPlayingMfxFromClExplosion` to the .cpp file so we don't have to include this here.